apps/shipments/filters.py

```diff
@@ -1,9 +1,14 @@
 from django_filters import rest_framework as filters
+from rest_framework_json_api.serializers import ValidationError
 
 from .models import Shipment, TransitState
 
 
 def filter_state(queryset, _, value):
+    # pylint:disable=protected-access
```

Comment:
As a general rule, if you have to disable a pylint rule to accomplish something, there is probably a better way to accomplish that thing.

Try the [EAFP](https://docs.python.org/3.6/glossary.html#term-eafp) approach - try initializing the enum with the value and if it fails, throw the error.

More docs here https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes